packages/trpc/server/routers/viewer/teams/inviteMember.handler.ts

```diff
@@ -81,7 +82,7 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
         from: ctx.user.name,
         to: input.usernameOrEmail,
         teamName: team.name,
-        joinLink: `${WEBAPP_URL}/signup?token=${token}&callbackUrl=/teams`,
+        joinLink: `${WEBAPP_URL}/signup?token=${token}&callbackUrl=/getting-started`, // we know that the user has not completed onboarding yet, so we can redirect them to the onboarding flow
```

Comment:
Previously we just redirected to teams -> Onboarding wasnt completed as it is a new user -> You get redirected to teams -> redirected to onboarding. Wasnt a nice flow for new users.

New:
New flow we just redirect to getting-started after signup to prevent flash